iothub/service/src/Feedback/AmqpFeedbackReceiver.cs

```diff
@@ -145,7 +145,7 @@ public override Task CompleteAsync(FeedbackBatch feedback)
                 _faultTolerantReceivingLink,
                 feedback.LockToken,
                 AmqpConstants.AcceptedOutcome,
-                true);
```

Comment:
All of these true -> false changes are for the "batchable" parameter in this method. We don't need to add or remove any logic in how we batch to make this change since it is all already in place.